platform/osgi/platform-osgi-conditions/src/main/java/org/codice/ddf/condition/SignerCondition.java

```diff
@@ -84,7 +86,9 @@ public boolean isPostponed() {
   @Override
   public boolean isSatisfied() {
     Map<X509Certificate, List<X509Certificate>> signerCertificates =
-        bundle.getSignerCertificates(Bundle.SIGNERS_TRUSTED);
+        AccessController.doPrivileged(
+            (PrivilegedAction<Map<X509Certificate, List<X509Certificate>>>)
+                () -> bundle.getSignerCertificates(Bundle.SIGNERS_TRUSTED));
```

Comment:
❓ Does this actually require doPrivileged? Doesn't look like getSignerCertificates throws a SecurityException

Reply:
Getting the signed content from the bundle can throw one.